src/simid/simid-client.js

```diff
@@ -314,6 +320,22 @@ export class SIMIDClient {
         return new SIMIDMessage(this._sessionId, messageId, type, args);
     }
 
+    _logMessage(prefix, msg) {
```

Comment:
If messages already log themselves (I'm assuming without using this `logMessage` method), should we rename this method `logDebugMessage` instead?

Reply:
Make sense, will do.